builds.yml

```diff
@@ -272,8 +272,10 @@ env:
   - SECURITY_ALERTS_API_ENABLED: ''
   # URL of security alerts API used to validate dApp requests
   - SECURITY_ALERTS_API_URL: 'http://localhost:3000'
```

Comment:
Why is this change not in the commit being cherry-picked?

Reply:
Same answer as here: https://github.com/MetaMask/metamask-extension/pull/28178/files#r1823521433